public/editor/scripts/project/project.js

```diff
@@ -1,4 +1,5 @@
 define(function(require) {
+  var $ = require("jquery");
```

Comment:
this should not be necessary: we already have jquery listed as a "known library" so you can instead say:
```
define(['jQuery'], function($) {
 ....
});
```
and have jQuery available that way.